app/src/test/java/org/simple/clinic/recentpatient/RecentPatientsScreenControllerTest.kt

```diff
@@ -141,7 +143,18 @@ class RecentPatientsScreenControllerTest {
     setupController()
     uiEvents.onNext(RecentPatientItemClicked(patientUuid = patientUuid))
 
+    verify(screen).updateRecentPatients(listOf(RecentPatientItem(
```

Comment:
Just a question, why does `updateRecentPatient` get triggered when we are opening the patient summary?

Reply:
Because the model gets rendered. This method should perhaps have been named better, like `displayRecentPatients`.

Reply:
Ah okay 👍